src/test/resources/features/Meeds/MeedsSpaces.feature

```diff
@@ -49,7 +49,7 @@ Feature: Meeds Space
     And I check that cancel button is displayed
     And I check that update button is displayed
 
-  @ignored
+
   Scenario: [SPC_MNG-8] Spaces applications management
     Given I am authenticated as admin
     When I go to the random space
```

Comment:
To avoid modifying shared random space settings, It would be better to create a new random space especially to this scenario that will not be reused after.

```suggestion
    When I create a random space
```

Reply:
done